cmd/earth/main.go

```diff
@@ -387,6 +395,72 @@ func newEarthApp(ctx context.Context, console conslogging.ConsoleLogger) *earthA
 				},
 			},
 		},
+		{
+			Name:   "org",
+			Usage:  "earthly organization administration",
```

Comment:
```suggestion
			Usage:  "Earthly organization administration",
```

Start with capital letter to stay consistent.

Same with a few other places below.